ember-app/app/models/new/issue.js

```diff
@@ -142,24 +169,18 @@ var Issue = Model.extend({
     }, function(){}, "json");
   },
   stateLabel: function(){
-    var state_label = this.get('other_labels').map((label)=> {
-        return label.name.toLowerCase();
-      }).find((name)=>{
-        return name.toLowerCase() === 'blocked' || name.toLowerCase() === 'ready';
-    });
-    return state_label || "";
+    return this.get('other_labels').map((label)=>{return label.name.toLowerCase()}).find((name)=>{
+      return name === 'blocked' || name === 'ready';
```

Comment:
Labels don't have their own their own intelligent models, so theres not really an efficient way other than checking every single label on its creation and setting the bool (as opposed to a computed property).